gravitee-apim-portal-webui-next/src/entities/user/custom-user-fields.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Definition of addition user registration fields
+ */
+export interface CustomUserFields {
```

Comment:
I'd suggest renaming this to singular

```suggestion
export interface CustomUserField {

Reply:
renamed